### PR TITLE
zh-translation: /zh/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus

### PR DESCRIPTION
**Pull request by nicole-lihui**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      - 'prometheus.{{INGRESS_DOMAIN}}'
+      labels:
+        cluster: '{{CLUSTER_NAME}}'
+{{< /text >}}
+
+注意：
+
+* `CLUSTER_NAME` 应该与创建集群时的值保持一致（通过 `values.global.multiCluster.clusterName` 设置）。
+
+* 没有开启 Promethus 端点验证。这意味着任何人都可以查询您的主集群的 Prometheus 实例，这是不可取的。
```
Comment (fleeto) [inline content/zh/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md:87]: Promethus -> Prometheus

File: content/zh/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+本教程的目的是为如何配置两个或者多个 Kubernetes 集群组成的 Istio 网格提供操作引导。这不是唯一的操作方式，而是演示一个使用 Prometheus 遥测多集群的可行方案。
+
+我们建议使用 Prometheus 监控 Istio 多集群是建立在 Prometheus [分层联合（Hierarchical Federation）](https://prometheus.io/docs/prometheus/latest/federation/#hierarchical-federation) 的基础上。
+通过 Istio 部署到每个集群中的 Prometheus 实例作为初始收集器，然后将数据聚合到网格层次的 Prometheus 实例上。网格层次的 Prometheus 既可以部署在网格之外（外部），也可以部署在网格内的集群中。
+
+## 安装 Istio 多集群{#multicluster-Istio-setup}
+
+按照[多集群安装](/zh/docs/setup/install/multicluster/)部分，在[多集群部署模型](/zh/docs/ops/deployment/deployment-models/#multiple-clusters)中选择可行的模型配置 Istio 多集群。为了能够实现本教程的目的，让示例都能够运行，并提出以下警告：
+
+**确保每个集群都安装了 Istio Prometheus 集群实例!**
```
Comment (kebe7jun) [inline content/zh/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md:23]: 两个集群？
确实是这样么？

File: content/zh/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+在每个集群中使用 Istio 独立部署的 Prometheus 是跨集群监控的基础，通过联合（Federation）的方式将Prometheus 的生产就绪实例在网格外部或在其中一个集群中运行。
+
+验证在每个集群中都运行一个 Prometheus 实例：
```
Comment (kebe7jun) [inline content/zh/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md:27]: 一个 的。

File: content/zh/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md (hunk 2 above)
Comment (nicole-lihui) [reply content/zh/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md:23]: 根据文档的意思，示例是多集群，其意思应该是至少两个或以上

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**